### PR TITLE
Content Dump!

### PR DESCRIPTION
**Pull request by SlideChaser**
Added a Nav Bar, JumboTron, and some sections about the singers!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,6 +10,101 @@
...
+
+        <div class="row justify-content-around">
+            <!-- Column 1 -->
+            <div class="col-2 text-center">
+                <img class="img-fluid" src="https://www.cheatsheet.com/wp-content/uploads/2020/01/BillieEilishtosingJamesBond.jpg" alt="">
+                <p class="py-2 font-weight-bold">
+                    Billy Eilish!!!
+                </p>
+                <p>
+                    Kind of like Michael Jackson but kinda less cra?
```
Comment (rdylina) [inline index.html:73]: Less what?

Comment (SlideChaser) [reply index.html:73]: Less Cra {cra=crazy}

**Review by rdylina — APPROVED** [review]
It's aight. 👍
Keep it PC for class though. lol

File: index.html (hunk 1 above)
Comment (rdylina) [reply index.html:73]: cray?

**Review by rdylina — COMMENTED** [review]
Appproved.

**Pull request closed without merging**